sdks/uniswapx-sdk/src/constants.ts

```diff
@@ -149,3 +149,17 @@ export const REVERSE_REACTOR_MAPPING: ReverseReactorMapping = Object.entries(
 export const BPS = 10000;
 
 export const MPS = BigNumber.from(10).pow(7);
+
+type PermissionedToken = {
+  address: string;
+  symbol: string;
+  usesProxy: boolean;
+}
```

Comment:
I opted not to include chainId as I couldn't imagine the same contract address on different chains having different implementations.

Reply:
ie. If BUIDL is at a different address on Unichain, then we would just add the Unichain address here as another entry.

Reply:
Hm I'd lean towards including it just to be cautious, there is an example of two different tokens having the same address on different chains

Reply:
Done!